client/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FormEvent } from 'react'
+import chameleonIcon from '/chameleon.svg'
+import searchIcon from '/search.svg'
+import personIcon from '/person-circle.svg'
+import '../../scss/_globals.scss'
+import { Navbar, Container, Nav, Col, NavDropdown, Form, Button} from 'react-bootstrap'
+
+
+const Header = () => {
+
+  const handleMobileSearchClick = () => {
+    console.log('do something when search icon is clicked in mobile view')
+    
+  }
+
+  const handleFormSubmit = () => {
+    console.log('do something when form is submitted from navbar on larger screen sizes')
+  }
+  
+  return (
+      <Container className='shadow' >
+        <Navbar sticky='top' >
+          <Container fluid>
+              <Navbar.Brand href="/" className='d-flex align-items-end'>
+                <img
+                  alt=""
+                  src={chameleonIcon}
+                  width="40"
+                  height="40"
+                  className="d-inline-block align-top me-3"
+                />
+                <h3 className='d-none d-md-inline-block text-primary font-weight-bolder mb-0'>Emerald Blog</h3>
```

Comment:
Title should be h1

Reply:
Hmm. I was trying to avoid h1 there since it's recommended to only have one h1 per page. Probably shouldn't be a heading tag at all. I was trying to make it bigger. Let me see about making a custom class and making it a good size that way since I have a better handle on how the react bootstrap stuff works now. Thank you!

Reply:
We can have this as the h1 and the others as h2's or leave the others as h1 and have this as a span like you did.